extension/neo4j/test/test_files/kleene_star_example.test

```diff
@@ -0,0 +1,19 @@
+-DATASET CSV empty
+-SKIP
+
+--
+
+-CASE MigrateFromNeo4jDocSampleUsingKleeneStar
+-STATEMENT load extension "${KUZU_ROOT_DIRECTORY}/extension/neo4j/build/libneo4j.kuzu_extension"
+---- ok
+-STATEMENT CALL NEO4J_MIGRATE("localhost", "neo4j", "czy990424", ["*"], ["*"]);
+---- ok
```

Comment:
- [ ] Test for output here. See `basic.test`. 
- [ ] Since you're using the `Neo4jDocSample` data, can you add a new create query to `neo4j.cypher`.
- [ ] Test the kleene star for the existing create query in `neo4j.cypher`.